src/main/scala/cromwell/binding/Workflow.scala

```diff
@@ -44,8 +44,8 @@ case class Workflow(name: String, declarations: Seq[Declaration], calls: Seq[Cal
    *         inputs that the user needs to provide to this workflow
    */
   def inputs: Seq[WorkflowInput] = {
-    val callInputs = for {call <- calls; input <- call.unsatisfiedInputs} yield input
-    val declarationInputs = for(declaration <- declarations; input <- declaration.asWorkflowInput) yield input
+    val callInputs = for { call <- calls; input <- call.unsatisfiedInputs } yield input
+    val declarationInputs = for { declaration <- declarations; input <- declaration.asWorkflowInput } yield input
```

Comment:
Personally I'm not a fan of semicolon-separating in for comprehensions, do the style police have an opinion on this?